### ISSUE TITLE
Biometric authentication setup fails on Tauri Windows app

### ISSUE THREAD

**Issue by andrejdaskalov** [padloc/padloc#600]
As the title describes, I cannot setup biometric authentication on the Tauri desktop app. It simply gives the following error: ```Biometric authentication failed! Canceling Setup. (Reason: The relying party ID is not a registrable domain suffix of, nor equal to the current domain.)```

**Comment by BrunoBernardino** [comment]
Thanks for reporting this @andrejdaskalov can you share a few more details?

- Is this on 4.1.0?
- Are you using Windows 10 or Windows 11?
- What's your biometric device (and perhaps actual computer device too)?

**Comment by BrunoBernardino** [comment]
Thanks @andrejdaskalov, I don't understand why we're getting this error, since `web.padloc.app` is exactly the RP ID (help @MaKleSoft?). Are you able to try the Electron app and see if that works? Also, I'm assuming you're connecting to Padloc's server, not your own (custom), right?

**Comment by MaKleSoft** [comment]
The relevant bit is this I think:

```
The relying party ID is not a registrable domain suffix of, nor equal to the current domain.
```

Basically the problem is that the Tauri app loads the the page from a local file instead of the `web.padloc.app` domain, which is causing the error here. Not sure how to work around this (or if it's even possible), but I'll take a look once I get the chance.

**Comment by andrejdaskalov** [comment]
> Thanks @andrejdaskalov, I don't understand why we're getting this error, since `web.padloc.app` is exactly the RP ID (help @MaKleSoft?). Are you able to try the Electron app and see if that works? Also, I'm assuming you're connecting to Padloc's server, not your own (custom), right?

@BrunoBernardino yes I'm using the released version, with Padloc's server. I'll try the Electron version though and I'll report back.

**Comment by andrejdaskalov** [comment]
@BrunoBernardino I installed electron but I am actually experiencing a different issue here: ```Biometric authentication failed! Canceling Setup. (Reason: The authenticator was previously registered)```.   Is this because I previously tried to set it up on Tauri? Or perhaps the issue lies within my device? 
I already tried revoking the device registered by Tauri, anything I can do to reset my device's settings for Padloc and set up biometric?

**Comment by MaKleSoft** [comment]
The electron app actually just loads the app from `https://web.padloc.app` which is why you're not seeing the same problem as with Tauri. You seem to have discovered a different problem though... looks like right now you can't register biometric auth for the web app from two different browsers (the electron app is basically treated like a browser here).

@andrejdaskalov Can you verify something for me? If you go remove disable all registered biometric authenticators in settings, does this resolve the problem with the Electron app?

**Comment by andrejdaskalov** [comment]
@MaKleSoft The laptop wasn't in the registered biometrics, only my phone was. I went ahead and removed it too, but it did not fix it (Same error). It seems the app isn't reporting a registered device, but when it tries to register it thinks it's already there. Maybe something got stuck server-side? 
Also if it helps I can build the app locally and provide you with output.
